Docs/MANUAL.md

```diff
@@ -0,0 +1,322 @@
+
+To create SSKR shares of your seed, go to the seed view, "Decrypt" the private data, and choose the Export icon at the top right. Then select the "Export as SSKR Multi-Share" option. This will allow you to set a number of groups, a number os shares, and thresholds You will usually leave the groups at one and just choose a number of shares (such as 3) and a threshold required to recover the seed (such as 2). 
+
+You can also choose a more complex methodology with SSKR's "groups", which allow you to define multiple groups, then to set a threshold of a certain number of shares from a certain number of groups: for example, you could create 3 groups, with a group threshold of 2, then have each group include 3 shares, with a threshold of 2. You'd then be able to recover your seed from 4 of the 9 shares, as long as 2 each come from 2 different groups. (But, this is more complex than most people will need: just do something simple like a 2 of 3 sharding or a 3 of 5 sharding if the group sharding sounds like something you wouldn't know how to use).
+
+When you export your shards, you can copy them to your clipboard as Bytewords (which are human-readable words) or `ur:crypto-sskr` (which are specially formatted URs that can be easily and reliably  read into any apps following the UR specification). However, the most useful means to export your SSKR may be by printing them. After you print them, you can cut out strips of paper for each share and then hand them to the people who will be storing them. We suggest asking them to store the QR code in [**Gordian QR Tool**](https://apps.apple.com/us/app/gordian-qr-tool/id1506851070) and then thoroughly destroy the slip of paper.
+
+<div align="center">
+  <table border=0>
+    <tr>
+      <td>
+        <a href="https://raw.githubusercontent.com/BlockchainCommons/GordianSeedTool-iOS/shannona-minimanual/images/st-sskr-export-1.jpeg"><img src="https://raw.githubusercontent.com/BlockchainCommons/GordianSeedTool-iOS/shannona-minimanual/images/st-sskr-export-1.jpeg" width=250></a> 
+        <br><div align="center"><b>SSKR Creation</b></div>
+      </center></td>
+      <td>
+        <a href="https://raw.githubusercontent.com/BlockchainCommons/GordianSeedTool-iOS/shannona-minimanual/images/st-sskr-export-2.jpeg"><img src="https://raw.githubusercontent.com/BlockchainCommons/GordianSeedTool-iOS/shannona-minimanual/images/st-sskr-export-2.jpeg" width=250></a> 
+        <br><div align="center"><b>SSKR Export</b></div>
+      </center></td>
+      <td>     
+        <a href="https://raw.githubusercontent.com/BlockchainCommons/GordianSeedTool-iOS/shannona-minimanual/images/st-sskr-expor-3.jpeg"><img src="https://raw.githubusercontent.com/BlockchainCommons/GordianSeedTool-iOS/shannona-minimanual/images/st-sskr-expor-3.jpeg" width=250></a> 
+        <br><div align="center"><b>SSKR Print</b></div>
+      </center></td>
+    </tr>
+  </table>
+</div>
+
```

Comment:
We may need a new section here. When someone requests the return of an SSKR shard, there is a threat which Shamir that anyone can monitor all communications, they can get a quorum and use the shard before you can (and of course if they've compromised the computer they all arrive on, but we have selected to ignore that threat). Thus before you return a shard you should confirm synchronously visual or voice (as no deep-fakes are fast) that you are returning the the right person, and you should not use a plaintext channel like mail or SMS, and instead use a secure channel like Signal. For the party requesting shards back, diversity of channels is good so if one channel is compromised, they don't get a quorum of shards.

Reply:
@ChristopherA "No deep fakes are fast."

...yet.

Reply:
Made some brief notes and added yet another threat